app/javascript/components/close_contact/CloseContact.js

```diff
@@ -288,11 +288,9 @@ class CloseContact extends React.Component {
     return (
       <React.Fragment>
         {!this.props.close_contact.id && (
-          <div className="pl-2">
-            <Button onClick={this.toggleModal}>
-              <i className="fas fa-plus"></i> Add New Close Contact
-            </Button>
-          </div>
+          <Button onClick={this.toggleModal}>
+            <i className="fas fa-plus fa-fw"></i> Add New Close Contact
```

Comment:
Gah sorry it's still slightly off, I also just realized the other two buttons have the button text in a span
```suggestion
            <i className="fas fa-plus fa-fw"></i>
            <span className="ml-2">Add New Close Contact</span>
```

Reply:
Alternatively, if you prefer it without the span, you could remove from Laboratory and Vaccination

Reply:
I like the additional spacing so I added that here: https://github.com/SaraAlert/SaraAlert/pull/746/commits/b092656131a49bb31b13f572be70531345b22c1c